gears/flywheel/pydeface-gear.json

```diff
@@ -0,0 +1,109 @@
+{
+  "name": "pydeface-gear",
+  "label": "Pydeface Gear",
+  "description": "A gear to remove facial structure from MRI images.",
+  "author": "Flywheel",
+  "maintainer": "Flywheel <support@flywheel.io>",
+  "license": "MIT",
+  "url": "https://github.com/flywheel-apps/test-gear",
+  "source": "https://github.com/flywheel-apps/pydeface-gear",
+  "cite": "https://github.com/poldracklab/pydeface",
+  "version": "0.1.0",
```

Comment:
Once the build is complete, this should be updated to reflect the tag and version build on DH - `1.0.0`